documentation/modules/evasion/windows/applocker_evasion_presentationhost.md

```diff
@@ -0,0 +1,34 @@
+## Intro
+
+This module is designed to evade solutions such as software restriction policies and Applocker.
+Applocker in its default configuration will block code in the form of executables (.exe and .com, .msi), scripts (.ps1, .vbs, .js) and dll's from running in user controlled directories.
+Applocker enforces this by employing whitelisting, in that code can only be run from the protected directories and sub directories of "Program Files" and "Windows"
+The main vector for this bypass is to use the trusted binary PresentationHost.exe to execute user supplied code as this binary is located within the trusted Windows directory.
+
+## Vulnerable Application
+
+This evasion will work on all versions of Windows that include .NET versions 3.5 or greater that has solutions such as Applocker or Software Restriction Policies active, that do not explicitly block PresentationHost.exe.
+
+## Options
+
+- **FILE_ONE** - Filename for the evasive file (default: presentationhost.xaml.cs).
+- **FILE_TWO** - Filename for the evasive file (default: presentationhost.manifest).
+- **FILE_THREE** - Filename for the evasive file (default: presentationhost.csproj).
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Do: `use evasion/windows/applocker_evasion_presentationhost`
+  3. Do: `set PAYLOAD <payload>` (note: only x86 payloads are supported by this module)
+  4. Do: `run`
+  5. The module will now display instructions of how to proceed
```

Comment:
You probably want to take everything below here (steps 6-11) and put it in its own section